test/functional/rpc_psbt.py

```diff
@@ -286,7 +357,120 @@ def run_test(self):
         # Unload extra wallets
         for i, signer in enumerate(signers):
             self.nodes[2].unloadwallet("wallet{}".format(i))
+    """
+
+    def run_ca_tests(self):
```

Comment:
For these tests let's put in some output-checking tests, making sure they're blinded(or not) based on the scenario. I think in this case we can just check that *all* outputs aside from fee are blinded or not.

Reply:
Same with `walletcreatefundedpsbt`, just make sure an expected-to-be-blinded psbt ends up getting blinded...

Reply:
Done!